docs/source/athena-setup.rst

```diff
@@ -49,13 +49,30 @@ Next, create the ``streamalert`` database:
 
   $ python manage.py athena create-db
 
-Finally, create the ``alerts`` table for searching generated StreamAlerts:
+Create the ``alerts`` table for searching generated StreamAlerts:
 
 .. code-block:: bash
 
   $ python manage.py athena create-table --type alerts --bucket <s3.bucket.id.goes.here>
 
+Create tables for data sent to StreamAlert:
+
+.. code-block:: bash
+
+  $ python manage.py athena create-table \ 
+    --type data \
+    --bucket <prefix>.streamalert.data \
+    --refresh_type add_hive_partition \
+    --table_name <log_name>
+
+Note: The log name above is representative of an enable log source to your StreamAlert deployment.
+
+For example, if you have cloudwatch in your sources, you would want to create tables for all possible subtypes.  This includes ``cloudwatch_events`` and ``cloudwatch_flow_logs``.  Also notice that ``:`` is substituted with ``_``; this is due to Hive limitations on table names.
```

Comment:
`if you have Cloudwatch in your sources`

Reply:
Include a link to what other characters are not allowed in a table name.

Reply:
you actually would not stylize `CloudWatch`, because that's not a valid schema name (it's case sensitive)

Reply:
maybe quote it if it's representing a raw string?

Reply:
@ryandeivert that works for me